addons/mail/static/src/js/client_action.js

```diff
@@ -663,6 +673,118 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
     },
 
     //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Updating control panel buttons visibility based on channel type
+     *
+     * @private
+     * @param {Object} channel
+     */
+    _updateControlPanelButtons: function (channel) {
```

Comment:
functions in private section should be ordered alphabetically, so this should go to the bottom